src/openrct2/network/network.cpp

```diff
@@ -784,6 +785,9 @@ void Network::LoadGroups()
 		}
 		json_decref(json);
 	}
+
+	// Host group should always contain all permissions.
+	group_list.at(0)->ActionsAllowed.fill(0xFF);
```

Comment:
The 0 is clear, this returns the first in the group which is the server I assume, ad `0xFF` sets all flags or something?